### DISCUSSION TITLE
Added: LDM notes for February 22nd, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4457] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2021/LDM-2021-02-22.md

## Agenda

1. Global `using`s
2. `using` alias improvements

**Comment by qrli** [comment]
Does it mean `global using static System.Math` will be supported?
Not sure good or bad...

**Reply by 333fred** [reply]
Yes, that will be supported.
